tests/huffman_test.c

```diff
@@ -340,6 +340,28 @@ static int test_huffman_decoder_partial_output(struct aws_allocator *allocator,
     return AWS_OP_SUCCESS;
 }
 
+AWS_TEST_CASE(huffman_decoder_allow_growth, test_huffman_decoder_allow_growth)
```

Comment:
Is there already a test case that verifies failure when growth is disabled?

Reply:
yes, the previous test in the file `huffman_decoder_partial_output` tests the growth-disabled behavior